File: QViz.NET/Periscope.QViz/JSON/APIBody.cs
using System;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for the Test API Body Details
	/// </summary>
	public class APIBody
	{
		/// <summary>
		/// Identifier of the API Body Object
		/// </summary>
		public string apiBodyId;

		/// <summary>
		/// Serial Number of the API Body Object
		/// </summary>
		public int srNo;

		/// <summary>
		/// Identifier of the linked API Object
		/// </summary>
		public string apiId;

		/// <summary>
		/// Identifier of the linked API Body Value Object
		/// </summary>
		public string bodyValueId;

		/// <summary>
		/// Date and Time on which this Model was Created
		/// </summary>
		public DateTime createdOn;

		/// <summary>
		/// Date and Time on which this Model was last Updated
		/// </summary>
		public DateTime updatedOn;

		/// <summary>
		/// Value of the API Body Object
		/// </summary>
		public APIBodyValue bodyValue;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public APIBody()
		{
			this.apiBodyId = "";
			this.srNo = 0;
			this.apiId = "";
			this.bodyValueId = "";
			this.createdOn = DateTime.Now;
			this.updatedOn = DateTime.Now;
			this.bodyValue = new APIBodyValue();
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/APIBody.java
package com.periscope.qviz.json;

import java.util.Date;

/**
 * JSON Model for the Test API Body Details
 */
public class APIBody {

	/**
	 * Identifier of the API Body Object
	 */
	public String apiBodyId;

	/**
	 * Serial Number of the API Body Object
	 */
	public Integer srNo;

	/**
	 * Identifier of the linked API Object
	 */
	public String apiId;

	/**
	 * Identifier of the linked API Body Value Object
	 */
	public String bodyValueId;

	/**
	 * Date and Time on which this Model was Created
	 */
	public Date createdOn;

	/**
	 * Date and Time on which this Model was last Updated
	 */
	public Date updatedOn;

	/**
	 * Value of the API Body Object
	 */
	public APIBodyValue bodyValue;

	/**
	 * Default Constructor
	 */
	public APIBody() {
		this.apiBodyId = "";
		this.srNo = 0;
		this.apiId = "";
		this.bodyValueId = "";
		this.createdOn = null;
		this.updatedOn = null;
		this.bodyValue = new APIBodyValue();
	}

}
